app/configs/config.py

```diff
@@ -86,7 +86,7 @@ def url_v2_download_pre(self) -> str:
 
     @computed_field
     def url_dataset_v2download(self) -> str:
```

Comment:
Could we use the format 
```suggestion
    def url_dataset_download_v2(self) -> str:
```